inspirehep/records/marshmallow/literature/base.py

```diff
@@ -223,6 +223,9 @@ class LiteratureESEnhancementV1(
     id_field = fields.Integer(dump_only=True, dump_to="id", attribute="control_number")
     thesis_info = fields.Nested(ThesisInfoSchemaForESV1, dump_only=True)
 
+    def get_ui_display(self, record):
+        return json.dumps(LiteratureMetadataUISchemaV1().dump(dict(record)).data)
```

Comment:
is the `dict` correct? don't you need data that's inside attributes of the record, like the citation count?

Reply:
nice catch 👍it's a left over